### ISSUE TITLE
Copay 0.9 wallet not accessible

### ISSUE THREAD

**Issue by kimmog** [bitpay/copay#3623]
I have a copay wallet made on version 0.9. The wallet is a text file in Lumia phone. When trying to import this wallet inTO copay 1.0 it is not successful as copay does not accept the password for this wallet. So I can not access the wallet and money in it. I tried this also in ubuntu by building up an environment for copay and tried to import this wallet there. The result is the same - password for wallet is rejected. The password should be ok; however it consists a special character "-". This should not  be a problem. Please advise.

**Comment by matiu** [comment]
Could you please detail (step-by-step) you how are trying to import the backup?

Please use the latest copay version (1.5.3).  Also, in order you check you backup and password, please go to https://bitwiseshiftleft.github.io/sjcl/demo/ enter your backup text in 'ciphertext' and entre the password and check if the password is ok.

**Comment by kimmog** [comment]
Hi,

When I put the ciphertext and password into this sjcl demo page, I get a 
msg: 'Can't decrypt: CORRUPT: ccm: tag doesn't match'

Does this mean, that my ciphertext is not correct?
As the phone is lumia, the ciphertext should be in onedrive (as a 
backup), but there is only an empty directory 
'CryptexIsolatedStorageFolder'. So I made a txt backup file (last April, 
when I created the wallet) on my lumia.

I attach the ciphertext file from lumia here, if you check if there are 
any structural errors etc in the text.

terveisin,

.kimmo.

On 09.12.2015 15:24, Matias Alejo Garcia wrote:
> Could you please detail (step-by-step) you how are trying to import the
> backup?
>
> Please use the latest copay version (1.5.3). Also, in order you check
> you backup and password, please go to
> https://bitwiseshiftleft.github.io/sjcl/demo/ enter your backup text in
> 'ciphertext' and entre the password and check if the password is ok.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitpay/copay/issues/3623#issuecomment-163229832>.
> file:///home/kimmo/wallet/copay.txt


{"iv":"9odPEL5b+x0jWyDiIGXIEQ==","v":1,"iter":10000,"ks":128,"ts":64,"mode":"ccm","adata":"","cipher":"aes","salt":"yMb31QoRr94=","ct":"k0RAWoii/9aA8YSjF04HgepeedFkAmQd7oz0YviQ9Q8bCia5dYWUbnsnDtJwad8Ti803GmIBIOpTX0fyOzVBMbGqsP3xMsNzDDxlb9aAoVoMxKv6LB3PAbg+nxZ1LTIqv7iUbSUOzobISkYjzDp+RNBDzOFJNJOhfKH9JGp5PnUdgyjHWk9dnOg3mE5LxRQZ3dxqzGlMyXmPoqVNArNpawQeM0uS/v54YhmmQJquY9QecPVChdkxDOr0ZDBffq2P85CPGfuPi5v9B5oHMZle//HdfnczC6KFYp4ADBQzK98dbpV1ScQc85GWjzBmC1xWgu4nJEGAytTroTP+j/55FgHv2Pp6qvkfhlyT5URozAzuZ7WFfGoW8BVGSMaQP1+hmvPh9fF3zro5F1oMwXFuQz6FZqg6CGGliESz0XBdlzTYj3XD42/dNT5kYVJjMDAV9WzlJGFuzB8i7x/tQxac6vZuErW1oaPOLXuYSGSswu61ssO/Tlp23TNDS+zqhSxuQAZC66RVnhfiJv6PhWZjAQGt888+C0BOUhWRlf9T0i22m9cFJHdWQVu6S77EkHwJBzpVFAXfe1hBQ//KyIXkHmTyaBM+fZo7pWIR9UMfF4IXAEtJZeDWXJOeR18vSBVvf6UTj6HGczgHQp8aGzajIHDu6oSkV1MXUdni3uHgljP3qj448lc1JByyAkM/YqIMSGRr6baReWNmziXUm16IjwjDCQ1hrSLSTd3L+XpAJEKMiW2pg7iCKKwkfU2kENOrzh2jOdjjV0zsEU1Zs0usfePx2ZHhLOOdET7cQprrWPsKZ2PXJnHdOgvW0NQlrgwbI1N0ic3pUS+9U+BdLz4AwfpRZ5ORH8PG0TXDp+v+iSCEpy38YCAJBz5XqNT+UBq28M8uQzAubhH8fN0SVn1ZnubTuNvmC4HC1/82wH6F8RqFuzhTaHHSSbSJiwzUETEfCN4xLBMKSniWAVi1eB+yJy01t0iXST8iA3jFgTsb1uK0A2W/yeNZ4XyDe4cA6VecThGaTiB8vcKdZElY5ioIZ5XGwREGOd4XyZrQmBnCq8BRJHTv1nEM4nK55o+vkMZxMmLbB3rfWKSdkdALjyu+2pzkhY8PkHF2rBrH8B471SlYSRcdixZgoMowDo4Qo72U0/6iw99W2lOsZYz9LeXReMGDTUOBwr7svvwIuNvCTR/SCCAhAmj826jvw6Qrym9ek0Em4atJ8IaOVOCQLC+Isjaw8e4zRj0ucHxtKLQsjnmEqjYpTU96gfDoblizgoWEUdyu/33Mr0lnZpWIZ4QcF8OEYgY26MEHz4bpMtgireqdznYaDsGsA507mjTIzOJt9XLOzQ+SHwc11dbNg2iYglNW87kJ7ZTuTaXTK3f4ssbCxUM50wROIeEO"}
